src/platform/user/authentication/utilities.js

```diff
@@ -39,11 +39,39 @@ export function clearRavenLoginType() {
   Raven.setTagsContext({ loginType: undefined });
 }
 
+function redirectWithGAClientId(redirectUrl) {
+  try {
+    let clientId;
+
+    // eslint-disable-next-line
```

Comment:
I would like to specify the rule being disabled when possible. In this case, adding a `no-undef`.